episodes/data.Rmd

```diff
@@ -110,12 +110,15 @@ Tip: if not, make sure that you activated `Generate documentation with Roxygen`
 Sometimes you need to use data in formats other than `.rda`.
 Examples of this are `.csv` or `.txt` files.
 
-In order to store raw data in your package, you have to follow a slightly different procedure.
-Namely:
+In order to store raw data in your package, you have to save them in `inst/extdata`.
+For example, we can add our example names vector here as a text file:
 
-1. Create a folder `inst/extdata/`, and save files there. Note that a user will have access to these data.
-2. When loading the data, do not describe the path as you usually would. Instead, use something like:
+```{r}
+dir.create("inst/extdata", recursive=TRUE)
+writeLines(example_names, "inst/extdata/names.txt")
```

Comment:
```suggestion
writeLines(example_names, "inst/extdata/names.csv")
```

Reply:
This doesn't have delimited fields, what makes you want this to be called a csv? Is it because we used `read.csv` later? I would be tempted to replace that with `readLines` anyway.

Reply:
@multimeric I don't think that it really matters whether it is `.txt` or `.csv` . What is important is that we use the same filename here and when reading in the file. I'm fine with changing `read.csv` with `readLines` and using `.txt` instead of `.csv`!